go.mod

```diff
@@ -22,3 +22,5 @@ require (
 	golang.org/x/text v0.23.0 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20250115164207-1a7da9e5054f // indirect
 )
+
+replace github.com/osrg/gobgp/v3 => ../gobgp
```

Comment:
Please remove this line `replace github.com/osrg/gobgp/v3 => ../gobgp.`

However, if someone wants to use a local version for testing Service Segment, this line is necessary.
Could you please mention in the "How is This Tested?" section of the PR description that this line should be added?